app/Meta.py

```diff
@@ -132,40 +130,32 @@ def insert_sites(fileName,shp_file,dbf_file,prj_file,shx_file):
 
         #Reading the shapefile as DataFrame
         data_g1=gpd.read_file(shp_file_target)              
-        
-        #data['geometry'] = data.geometry.astype(str)
-        
-        """for i in range(data_g.shape[0]):
-            p=data_g.iat[i,10]
-            geos.lgeos.GEOSSetSRID(p._geom, 4326)
-            data.iat[i,6]=p.wkb_hex"""
 
         data_g  = data_g1.to_crs({'init': 'epsg:4326'})
 
-        flat_list = []
         for index, row in data_g.iterrows():
+            flat_list = []
             for pt in list(row['geometry'].exterior.coords):
                 pt=pt+(115,)
                 flat_list.append(pt)
             poly = Polygon(flat_list)
             data.loc[index, 'geometry'] = poly
         
         if(all(x in accepted_columns for x in columns)):
-            data['notes']=data_g['notes']
-            #data['geometry']=data_g['geometry']
             data['time_zone'].fillna("America/Phoenix", inplace = True)
             data['soilnotes'].fillna("some text", inplace = True)      
```

Comment:
```suggestion
            data['soilnotes'].fillna("", inplace = True)      
```

convention in the database is empty string